docs/source/migrate_experiment_tracking.md

```diff
@@ -0,0 +1,140 @@
+# Migration from Kedro-Viz Native Experiment Tracking to Kedro-MLflow
+
+With the deprecation of Kedro-Viz experiment tracking from version 11.0.0, transitioning to [Kedro-MLflow](https://kedro-mlflow.readthedocs.io/en/stable/) offers enhanced experiment tracking and artifact management. This guide outlines the steps to:
+
+1. **Remove deprecated configurations related to Kedro-Viz experiment tracking.**
+2. **Update existing dataset configurations in the catalog.**
+3. **Optionally, delete old experiment tracking data.**
+4. **Refer to MLflow documentation for further setup.**
+
+## Step 1: Remove deprecated configurations related to Kedro-Viz experiment tracking
+
+### a. Remove Session Store Setup
+
+In `src/<project_name>/settings.py`, locate and remove the session store configuration:
+
+```python
+from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
+
+SESSION_STORE_CLASS = SQLiteStore
+SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}
+```
+
+### b. Remove Environment Variables
+Remove any environment variables associated with collaborative tracking:
+
+```bash
+export KEDRO_SQLITE_STORE_USERNAME="your_unique_username"
+```
+
+### c. Delete SQLite Database
+If a session_store.db SQLite database was created, delete it. Typically, it is located in the project root or under the data directory.
```

Comment:
```suggestion
If a `session_store.db` SQLite database was created, delete it. Typically, it is located in the project root or under the data directory.
```

Can't it also be under `.viz` ? https://github.com/kedro-org/kedro-viz/issues/1951